docs/topics/wildfly-swarm/docs/reference/fractions/elytron.adoc

```diff
@@ -0,0 +1,996 @@
+The principal represented by the KeyTab
+
+swarm.elytron.kerberos-security-factories._KEY_.relative-to:: 
+The name of another previously named path, or of one of the standard paths provided by the system. If 'relative-to' is provided, the value of the 'path' attribute is treated as relative to the path specified by this attribute.
+
+swarm.elytron.kerberos-security-factories._KEY_.request-lifetime:: 
+How much lifetime (in seconds) should be requested for newly created credentials.
+
+swarm.elytron.kerberos-security-factories._KEY_.required:: 
+Is the keytab file with adequate principal required to exist at the time the service starts?
+
+swarm.elytron.kerberos-security-factories._KEY_.server:: 
+If this for use server side or client side?
+
+swarm.elytron.kerberos-security-factories._KEY_.wrap-gss-credential:: 
+Should generated GSS credentials be wrapped to prevent improper disposal or not?
+
+swarm.elytron.key-managers._KEY_.algorithm:: 
+The name of the algorithm to use to create the underlying KeyManagerFactory.
+
+swarm.elytron.key-managers._KEY_.alias-filter:: 
+A filter to apply to the aliases returned from the KeyStore, can either be a comma separated list of aliases to return or one of the following formats ALL:-alias1:-alias2, NONE:+alias1:+alias2
+
+swarm.elytron.key-managers._KEY_.credential-reference:: 
+The credential reference to decrypt KeyStore item. (Not a password of the KeyStore.)
+
+swarm.elytron.key-managers._KEY_.key-store:: 
+Reference to the KeyStore to use to initialise the underlying KeyManagerFactory.
+
+swarm.elytron.key-managers._KEY_.provider-name:: 
+The name of the provider to use to create the underlying KeyManagerFactory.
+
+swarm.elytron.key-managers._KEY_.providers:: 
+Reference to obtain the Provider[] to use when creating the underlying KeyManagerFactory.
+
+swarm.elytron.key-store-realms._KEY_.key-store:: 
+Reference to the KeyStore that should be used to back this security realm.
+
+swarm.elytron.key-stores._KEY_.alias-filter:: 
+A filter to apply to the aliases returned from the KeyStore, can either be a comma separated list of aliases to return or one of the following formats ALL:-alias1:-alias2, NONE:+alias1:+alias2
+
+swarm.elytron.key-stores._KEY_.attribute-synchronized:: 
+The time this KeyStore was last loaded or saved. Note: Some providers may continue to apply updates after the KeyStore was loaded within the application server.
+
+swarm.elytron.key-stores._KEY_.credential-reference:: 
+The reference to credential stored in CredentialStore under defined alias or clear text password.
+
+swarm.elytron.key-stores._KEY_.loaded-provider:: 
+Information about the provider that was used for this KeyStore.
+
+swarm.elytron.key-stores._KEY_.modified:: 
+Indicates if the in-memory representation of the KeyStore has been changed since it was last loaded or stored.  Note: For some providers updates may be immediate without further load or store calls.
```

Comment:
Reformat:

> Note: For some providers updates may be immediate without further load or store calls.

to either be a note or not use `Note: `

Reply:
Note to self: not easily fixable in our sources.